docs/SetupGuide_DotnetCSharpScript.md

```diff
@@ -52,11 +52,11 @@ These instructions will guide you through creating your Function Project and add
 
 4. Enable SQL bindings on the csx function created above. More information can be found in the [Azure SQL bindings for Azure Functions docs](https://aka.ms/sqlbindings).
 
-    Update the `host.json` file inside **MyApp/MyCSXFunction/** to the preview extension bundle.
+    Update the `host.json` file inside **MyApp/MyCSXFunction/** to the extension bundle.
```

Comment:
nit: Above you changed to `with the extension bundle`, while here you're leaving it as `to the extension bundle`. It'd be good to be consistent.

My suggestion (with added stuff like the specific folder such as this example added in as needed)

`In the 'host.json' file add or update the extensionBundle version`